### DISCUSSION TITLE
Error meshcentral

### DISCUSSION THREAD

**Discussion by Iberid1974** [amidaware/tacticalrmm#2033] [General]
Thank you in advance.
How do I solve this central mesh problem? Since when entering the url I confused the . with the -.
The current address is mesh.rmm.iltecnico.site
and it should be mesh.rmm-iltecnico.site 
and now the mesh part, the connection via take control, does not work.
Thank you very much again
![2024-10-16 17_58_19-Greenshot](https://github.com/user-attachments/assets/dab8bd85-f4aa-4254-9536-5942753cb9b9)

**Comment by dinger1986** [comment]
Best to just wipe and reinstall and don't use --insecure as you are using a real domain

**Reply by Iberid1974** [reply]
Gracias, la solución fue esa !! ahora estoy buscando como vincularlo a una solucion de tickets
